lib/shoryuken/options.rb

```diff
@@ -30,9 +30,10 @@ def active_job?
         defined?(::ActiveJob)
       end
 
-      def add_group(group, concurrency)
+      def add_group(group, concurrency, delay: 0)
         groups[group] ||= {
           concurrency: concurrency,
+          delay: delay,
```

Comment:
> Sounds great. Let me know about lib/shoryuken/environment_loader.rb. I think I'm missing where it parses the YML variables to add the delay into the group.

@lucas-ibotta sure. I think [here](https://github.com/phstc/shoryuken/blob/master/lib/shoryuken/environment_loader.rb#L133), you can:

```ruby
Shoryuken.add_group(group, options.fetch(:concurrency, 25), delay: options.fetch(:delay, 0))
```

Reply:
[Here](https://github.com/phstc/shoryuken/blob/bc7938942270a1171d1597064698d88c3f36a4ec/lib/shoryuken/polling/base.rb#L63) you also need to get the delay from the group instead of the global one.

[This](https://github.com/phstc/shoryuken/blob/352b20642b4b8a123b821228231ed5a5c618a9cc/lib/shoryuken/options.rb#L79) is an example how we retrieve `polling_strategy` per group. You can do something similar for `delay`.

Does that make sense?

For testing that locally, you can create two groups with a 25 seconds delay and the other with the default delay. Then play in the console: 

```sh
aws sqs send-message --message-body "Test" --queue-url https://sqs.us-east-1.amazonaws.com/.../...
```

WDYT?

Reply:
Yup, makes sense. I'll start working on that soon and push the changes. 😄